### DISCUSSION TITLE
Create a new request

### DISCUSSION THREAD

**Discussion by DrMarkX** [Snow-Shell/servicenow-powershell#233] [Q&A]
I'm having an issue creating a request that is not an incident or non-catalog.  I created a variable that stores the minimum required criteria to create a request and used this command.
`New-ServiceNowRecord -Table 'sc_req_item' -Values $minCriteria`
This gave me a 403 Forbidden error and when I changed the table to 'sn_cn' I got a 400 Bad Request error.
The request should create an RITM and the subsequent Task to our team. Any guidance on this would be appreciated.

**Comment by gdbarron** [comment]
Are you sure you have the correct permissions to do this?  I was able to execute `New-ServiceNowRecord -table 'sc_req_item' -Values @{'short_description'='test1'}` without issues.  If you run this with -PassThru it will provide back the resulting object.  Take the `number` and provide it as the request_item for the subsequent task creation.

```
$newRitm = New-ServiceNowRecord -table 'sc_req_item' -Values @{'short_description'='test1'} -PassThru
$newTask = New-ServiceNowRecord -table 'sc_task' -Values @{'request_item'=$newRitm.number; 'short_description'='mytask1'} -PassThru
```
